### ISSUE TITLE
Question about the requirement of GPU

### ISSUE THREAD

**Issue by Hugo-cell111** [cskyl/SAM_WSSS#7]
Hi! How many GPUs and what kind of GPUs are required in your work? Thanks!

**Comment by cskyl** [comment]
Hi, thank you for asking. You don't need any GPU to generate the pseudo labels, they can be fully produced using only CPUs. 
On a single process, the rate is around 100~200 imgs / minute on my laptop, by using multi process, it takes about 10mins to generate the pseudo label for PASCAL train_aug (~10k imgs).

> Hi! How many GPUs and what kind of GPUs are required in your work? Thanks!

**Comment by JiaojiaoYe1994** [comment]
> Hi, thank you for asking. You don't need any GPU to generate the pseudo labels, they can be fully produced using only CPUs. On a single process, the rate is around 100~200 imgs / minute on my laptop, by using multi process, it takes about 10mins to generate the pseudo label for PASCAL train_aug (~10k imgs).
> 
> > Hi! How many GPUs and what kind of GPUs are required in your work? Thanks!

Hi! How many GPUs and how long are required for training semantic segmentation with pseudo labels? Thank you

**Comment by cskyl** [comment]
> > Hi, thank you for asking. You don't need any GPU to generate the pseudo labels, they can be fully produced using only CPUs. On a single process, the rate is around 100~200 imgs / minute on my laptop, by using multi process, it takes about 10mins to generate the pseudo label for PASCAL train_aug (~10k imgs).
> > > Hi! How many GPUs and what kind of GPUs are required in your work? Thanks!
> 
> Hi! How many GPUs and how long are required for training semantic segmentation with pseudo labels? Thank you

Hi! We are using 2 V100 GPUs to train the semantic segmentation models with our pseudo labels, more details about the training are following the repo of Deeplab Model https://github.com/kazuto1011/deeplab-pytorch. 
You can find the hyperparameters we used in our paper https://arxiv.org/pdf/2305.05803.pdf.
Thank you for your comments!
